app/mailers/producer_mailer.rb

```diff
@@ -36,6 +36,7 @@ def load_data
     line_items = line_items_from(@order_cycle, @producer)
 
     @grouped_line_items = line_items.group_by(&:product_and_full_name)
+    @ready_for = @order_cycle.pickup_time_for(@coordinator)
```

Comment:
Ahh... we might need to be a bit careful here. Pickup times are on outgoing exchanges, and I think it's possible that the OC's **coordinator** enterprise might not always be the enterprise handling the outgoing exchange...?

Reply:
This is a tricky one. Maybe @filipefurtad0 has some insight?

Reply:
oh yes, that's right :see_no_evil:

Reply:
That's a great catch @Matt-Yorkley - I think the key role here would be the `distributor`, instead of the coordinator. As I understand this, a coordinator can have different distributors, each with different `pickup_time`s. So, each "Notify Producers" email will may have different `pickup_time` or `ready for` info.

Reply:
Thanks, good to know. I have updated this [here ](https://github.com/openfoodfoundation/openfoodnetwork/pull/6674/commits/70a557cea37eea43b836c1f9d3d57730150c1961) so it displays the pickup times for each distributor now but not sure if it's correct yet because I'm not very familiar with incoming/outgoing exchanges.